Задача 3. Исследовалось влияние препарата на уровень давления пациентов. Сначала
измерялось давление до приема препарата, потом через 10 минут после приёма препарата. Есть
ли статистически значимые различия между измерениями давления? В выборках не соблюдается условие нормальности.
1е измерение до приема препарата: 150, 160, 165, 145, 155
2е измерение через 10 минут: 140, 155, 150, 130, 135

In [3]:
import numpy as np
from scipy import stats

In [4]:
sample1 = np.array([150, 160, 165, 145, 155]) 
sample2 = np.array([140, 155, 150, 130, 135])

Принимаем α=0,05.

In [5]:
stats.wilcoxon(sample1, sample2)

WilcoxonResult(statistic=0.0, pvalue=0.0625)

Т.к. pvalue=0.0625 > α=0,05, можно говорить о том что статистически значимых различий между измерениями давления нет.